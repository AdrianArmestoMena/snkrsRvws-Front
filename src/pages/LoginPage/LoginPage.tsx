import Login from "../../components/LogIn/LogIn";
import Formstyle from "../../stylesUtils/Forms.style";

const LoginPage = (): JSX.Element => {
  return (
    <Formstyle>
      <h2 className="action-call">
        Join the {<span className="action-call__outstanding">SnkrsRvws</span>}{" "}
        community
      </h2>
      <Login />
    </Formstyle>
  );
};

export default LoginPage;
