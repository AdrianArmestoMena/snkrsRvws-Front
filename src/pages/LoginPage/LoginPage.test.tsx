import TestRenderer from "react-test-renderer";
import Wrapper from "../../test-utils/Wrapper";
import LoginPage from "./LoginPage";

describe("Given a Login Page component", () => {
  describe("When it's instantiated", () => {
    test("Then it should match the html from Home page", () => {
      const expectedLoginPage = TestRenderer.create(
        <Wrapper>
          <LoginPage />
        </Wrapper>
      );

      expect(expectedLoginPage).toMatchSnapshot();
    });
  });
});
