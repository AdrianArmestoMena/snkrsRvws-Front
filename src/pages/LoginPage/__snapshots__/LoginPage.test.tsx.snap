// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Given a Home Page component When it's instantiated Then it should match the html from Home page 1`] = `
<main
  className="sc-bczRLJ kbzsDj"
>
  <h2
    className="action-call"
  >
    Join the 
    <span
      className="action-call__outstanding"
    >
      SnkrsRvws
    </span>
     
    community
  </h2>
  <form
    className="form"
    noValidate={true}
    onSubmit={[Function]}
  >
    <div
      className="mb-3"
    >
      <label
        className="form-label"
        htmlFor="Name"
      >
        User Name
      </label>
      <input
        className="form-control"
        id="Name"
        name="userName"
        onChange={[Function]}
        placeholder="Enter your name"
        required={true}
        type="text"
        value=""
      />
      <div
        className="invalid-feedback"
      >
        Please choose a username.
      </div>
    </div>
    <div
      className="mb-3"
    >
      <label
        className="form-label"
        htmlFor="Pasword"
      >
        Password
      </label>
      <input
        className="form-control"
        id="Pasword"
        name="password"
        onChange={[Function]}
        placeholder="Password"
        required={true}
        type="password"
        value=""
      />
      <div
        className="invalid-feedback"
      >
        Please provide a password.
      </div>
    </div>
    <button
      className="col mt-4 btn btn-primary form__button btn btn-primary"
      disabled={false}
      type="submit"
    >
      Login
    </button>
  </form>
</main>
`;

exports[`Given a Login Page component When it's instantiated Then it should match the html from Home page 1`] = `
<main
  className="sc-bczRLJ kbzsDj"
>
  <h2
    className="action-call"
  >
    Join the 
    <span
      className="action-call__outstanding"
    >
      SnkrsRvws
    </span>
     
    community
  </h2>
  <form
    className="form"
    noValidate={true}
    onSubmit={[Function]}
  >
    <div
      className="mb-3"
    >
      <label
        className="form-label"
        htmlFor="Name"
      >
        User Name
      </label>
      <input
        className="form-control"
        id="Name"
        name="userName"
        onChange={[Function]}
        placeholder="Enter your name"
        required={true}
        type="text"
        value=""
      />
      <div
        className="invalid-feedback"
      >
        Please choose a username.
      </div>
    </div>
    <div
      className="mb-3"
    >
      <label
        className="form-label"
        htmlFor="Pasword"
      >
        Password
      </label>
      <input
        className="form-control"
        id="Pasword"
        name="password"
        onChange={[Function]}
        placeholder="Password"
        required={true}
        type="password"
        value=""
      />
      <div
        className="invalid-feedback"
      >
        Please provide a password.
      </div>
    </div>
    <button
      className="col mt-4 btn btn-primary form__button btn btn-primary"
      disabled={false}
      type="submit"
    >
      Login
    </button>
  </form>
</main>
`;
